ee/dtr/admin/configure/external-storage/storage-backend-migration.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Switch storage backends
+description: Storage backend migration for Docker Trusted Registry
+keywords: dtr, storage drivers, local volume, NFS, Azure, S3,
+---
+
+Starting in DTR 2.6, switching storage backends initializes a new metadata store and erases your existing tags. This helps facilitate online garbage collection, which has been introduced in 2.5 as an experimental feature. In earlier versions, DTR would subsequently start a `tagmigration` job to rebuild tag metadata from the file layout in the image layer store. This job has been discontinued, as your storage backend could get out of sync with your DTR metadata, like your manifests and existing repositories. As best practice, DTR storage backends and metadata should always be moved, backed up, and restored together.
+
+## DTR 2.6.4 and above
+
+In DTR 2.6.4, a new flag, `--storage-migrated`, [has been added to `docker/dtr reconfigure`](/reference/dtr/2.6/cli/reconfigure/) which lets you indicate the migration status of your storage data during a reconfigure.  
+
+### Best practice for data migration
+
+Docker recommends the following steps for your storage backend and metadata migration:
+
+1. Disable garbage collection by selecting "Never" under **System > Garbage Collection**, so blobs referenced in the backup that you create continue to exist. See [Garbage collection](/ee/dtr/admin/configure/garbage-collection/) for more details.
```

Comment:
Is the best practice to disable garbage collection for the duration of the migration?
I'm guessing that is the intention, but it could be made more clear.